packages/e2e-tests/specs/performance/site-editor.test.js

```diff
@@ -71,7 +71,7 @@ describe( 'Site Editor Performance', () => {
 				}
 			} );
 
-			dispatch( 'core/block-editor' ).resetBlocks( blocks );
+			await dispatch( 'core/block-editor' ).resetBlocks( blocks );
```

Comment:
This change is a bit weird for me. AFAIK, this action is synchronous

Reply:
`resetBlocks()` returns a promise though. I think it's because it is a thunk: https://github.com/WordPress/gutenberg/blob/a770a6eaceacab0ae499950b4c539676662f4148/packages/block-editor/src/store/actions.js#L68-L73

But this is likely not the root cause of the failure. We can omit this in this PR.